Tests/CommandsTests/TestToolTests.swift

```diff
@@ -31,14 +31,15 @@ final class TestToolTests: XCTestCase {
         XCTAssert(try execute(["--version"]).contains("Swift Package Manager"))
     }
 
-    // Verifies that giving several options still works.
-    func testSanitizeThreadAddress() throws {
+    // Verifies that sanitization works
+    func testSanitizeThread() throws {
         fixture(name: "Miscellaneous/SanitizersTest") { path in
-            let cmdline = ["--sanitize=thread", "--sanitize=address"]
+            let cmdline = ["--sanitize=thread"]
             XCTAssertThrows(try execute(cmdline, packagePath: path)) { (error: SwiftPMProductError) -> Bool in
                 switch error {
-                case .executionFailure:
-                    return true
+                case .executionFailure(_, let stdout, let stderr):
+                    return stderr.range(of: "ThreadSanitizer:") != nil
+                        && stderr.range(of: "access race") != nil
```

Comment:
This doesn't run properly because the executor doesn't capture the test output properly.

Reply:
I think thats an actual bug. I see this in the crash report locally:

```
Application Specific Information:
dyld: launch, loading dependent libraries
DYLD_INSERT_LIBRARIES=:/path/to/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/clang/lib/darwin/libclang_rt.tsan_osx_dynamic.dylib

Dyld Error Message:
  could not load inserted library '' because image not found
```